crates/node_binding/src/hook.rs

```diff
@@ -32,7 +32,7 @@ pub enum Hook {
   AfterCompile,
   FinishModules,
   OptimizeModules,
-  OptimizaTree,
+  OptimizeTree,
```

Comment:
don't we have typo check now?

Reply:
`optimiza tree` can be detected but `OptimizaTree` can't, I guess